CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## UNRELEASED
 
+### Refactor
```

Comment:
I think this should be in the "Changed" category (if we want to follow [common changelog](https://common-changelog.org/) conventions), or, if we want to introduce a separate category for refactorings, it should be "Refactored"

Reply:
Thanks. I never find the resource back. Will change that